docs/config/apache2-site

```diff
@@ -18,13 +18,13 @@
 
 	DirectoryIndex main.php index.html
 
-	ErrorLog %LOGDIRECTORY%/%PROJECTNAME%-error.log
+	ErrorLog %LOGDIRECTORY%/error.log
```

Comment:
I'm curious about this change. I'm not very familiar with Apache config, but isn't there some value in project specific log files? This will apply to non-vagrant installs too, right?